content/en/docs/hertz/tutorials/basic-feature/route.md

```diff
@@ -0,0 +1,174 @@
+---
+title: "route"
+date: 2022-09-06
+weight: 2
+description: >
+
+---
+
+## Route Registration
+
+Hertz provides methods like `GET`, `POST`, `PUT`, `DELETE`, `ANY` for registering routes.
+
+Sample Code:
+```go
+package main
+
+import (
+	"context"
+	"github.com/cloudwego/hertz/pkg/app"
+	"github.com/cloudwego/hertz/pkg/app/server"
+	"github.com/cloudwego/hertz/pkg/protocol/consts"
+)
+
+func main(){
+	h := server.Default(server.WithHostPorts("127.0.0.1:8080"))
+	h.GET("/get", func(ctx context.Context, c *app.RequestContext) {
+		c.String(consts.StatusOK, "get")
+	})
+	h.POST("/post", func(ctx context.Context, c *app.RequestContext) {
+		c.String(consts.StatusOK, "post")
+	})
+	h.PUT("/put", func(ctx context.Context, c *app.RequestContext) {
+		c.String(consts.StatusOK, "put")
+	})
+	h.DELETE("/delete", func(ctx context.Context, c *app.RequestContext) {
+		c.String(consts.StatusOK, "delete")
+	})
+	h.PATCH("/patch", func(ctx context.Context, c *app.RequestContext) {
+		c.String(consts.StatusOK, "patch")
+	})
+	h.HEAD("/head", func(ctx context.Context, c *app.RequestContext) {
+		c.String(consts.StatusOK, "head")
+	})
+	h.OPTIONS("/options", func(ctx context.Context, c *app.RequestContext) {
+		c.String(consts.StatusOK, "options")
+	})
+	h.Spin()
+}
+
+```
+
+## Group
+
+Hertz provides the capability of `Group`, which are used to support route grouping functionality, and the middleware can also register with `Group`.
+
+
+Sample Code:
+```go
+package main
+
+import (
+	"context"
+	"github.com/cloudwego/hertz/pkg/app"
+	"github.com/cloudwego/hertz/pkg/app/server"
+	"github.com/cloudwego/hertz/pkg/protocol/consts"
+)
+
+func main(){
+	h := server.Default(server.WithHostPorts("127.0.0.1:8080"))
+	v1 := h.Group("/v1")
+	h.GET("/get", func(ctx context.Context, c *app.RequestContext) {
```

Comment:
there is a problem with the code here

Reply:
https://github.com/cloudwego/cloudwego.github.io/pull/348